indico/util/string.py

```diff
@@ -168,6 +168,7 @@ def process(self, html):
     'text-decoration', 'text-indent', 'text-shadow', 'text-transform', 'unicode-bidi', 'visibility', 'vertical-align',
     'width', 'widows', 'white-space', 'word-spacing', 'word-wrap', 'z-index'
 ]
+BLEACH_ALLOWED_PROTOCOLS_HTML = ['cid']  # used for embedded image
```

Comment:
This will most likely break the current list of allowed protocols since you're only passing cid but no http/https now.